txcache/constants.go

```diff
@@ -3,3 +3,4 @@ package txcache
 const diagnosisMaxTransactionsToDisplay = 10000
 const diagnosisSelectionGasRequested = 10_000_000_000
 const initialCapacityOfSelectionSlice = 30000
+const selectionLoopDurationCheckInterval = 16
```

Comment:
Selection loop timeout is checked each 16 iterations (more or less each 16 selected transactions) - as a minor optimization, since time getting might incur an OS call.

Switched to 10 (more "natural").

Reply:
ok!